src/main/java/nl/novi/techiteasy/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,95 @@
+package nl.novi.techiteasy.controllers;
+
+import nl.novi.techiteasy.exceptions.RecordNotFoundException;
+import nl.novi.techiteasy.exceptions.TelevisionNameTooLongException;
+import nl.novi.techiteasy.models.Television;
+import nl.novi.techiteasy.repositories.TelevisionRepository;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/api/televisions")
```

Comment:
Netjes om hier api in te gebruiken wat nog mooier is is api/v1/televisions Zo kan je altijd een update doen naar v2 in de toekomst

Reply:
Dat is klopt Thanks!